clinica/iotools/bids_utils.py

```diff
@@ -238,13 +239,24 @@ def create_sessions_dict(
                     session_names = get_bids_subjs_list(subj_dir)
                     for s in session_names:
                         s_name = s.replace("ses-", "")
+                        if study_name is not "OASIS3":
```

Comment:
Please use `==` or `!=` for string comparison. Although identity checking *may* work, it is an antipattern and can fail in some subtle ways.